src/Utils/utils.ts

```diff
@@ -544,3 +544,13 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+export const handleNegativeValue = (
+  event: React.FormEvent<HTMLInputElement>,
+): void => {
+  const input = event.currentTarget;
+  input.value = input.value.replace(/-/g, "");
+  if (isNaN(Number(input.value))) {
+    input.value = "";
+  }
```

Comment:
this just clears the input everytime a special character is pressed. Instead of clearing, I'd recommend cleaning the non numeric characters so that user doesn't loose what they've typed.

Eg: User types `12)`, instead of clearing when i type `)`, it should reset to just `12`

Reply:
Hey @rithviknishad, I updated the function logic, but it's clearing the value when entering '+/-' after any digit. Do you have any suggestions to improve the logic?
```js
export const sanitizeNumberInput = (
  event: React.FormEvent<HTMLInputElement>,
): void => {
  const input = event.currentTarget;
  input.value = input.value.replace(/[^0-9]/g, "");
};
```